Classes/Domain/Service/JsonService.php

```diff
@@ -0,0 +1,235 @@
+<?php
+namespace MASK\Mask\Domain\Service;
+
+use TYPO3\CMS\Core\Messaging\FlashMessage;
+use TYPO3\CMS\Core\Messaging\FlashMessageService;
+use TYPO3\CMS\Core\SingletonInterface;
+use TYPO3\CMS\Core\Utility\ArrayUtility;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Object\ObjectManager;
+use TYPO3\CMS\Extbase\Utility\DebuggerUtility;
+
+/**
+ * Class JsonService
+ * @package MASK\Mask\Domain\Service
+ */
+class JsonService implements SingletonInterface
+{
+    /**
+     * @var self|null
+     */
+    private static $_instance = null;
+
+    /**
+     * json configuration
+     * @var array
+     */
+    private $json = null;
+
+    /**
+     * @var ObjectManager
+     */
+    private $objectManager;
+
+    /**
+     * @return JsonService
+     */
+    public static function getInstance()
+    {
+        if (null === self::$_instance) {
+            self::$_instance = new self;
+
+            /* @var $message \TYPO3\CMS\Core\Messaging\FlashMessage */
+            self::$_instance->objectManager = GeneralUtility::makeInstance(ObjectManager::class);
+        }
+        return self::$_instance;
+    }
+
+    /**
+     * @param string $path
+     *
+     * @return array
+     */
+    public function getConfiguration($path = '')
+    {
+        if ($this->json !== null) {
+            return $this->json;
+        }
+
+        if (substr($path, -strlen('.json')) == '.json') {
```

Comment:
strlen('.json') is a known value - no need for a php function. use === for strict comparision